main.js

```diff
@@ -32,23 +32,37 @@ const printBoard = () => {
   console.log('2 ' + board[2].join(' | '));
 }
 
+
 const horizontalWin = () => {
-  // Your code here to check for horizontal wins
+  if (board[0][0] + board[0][1] + board[0][2] === 'XXX' || board[0][0] + board[0][1] + board[0][2] === 'OOO') {return true} 
+  else if (board[1][0] + board[1][1] + board[1][2] === 'XXX' || board[1][0] + board[1][1] + board[1][2] === 'OOO') {return true}
+  else if (board[2][0] + board[2][1] + board[2][2] === 'XXX' || board[2][0] + board[2][1] + board[2][2] === 'OOO') {return true}
+  else {return false}
```

Comment:
as long as you know that this can also be written as:
`if () {`
`    return;`
`} else if ( ) {`
`   return`
`}`
then i have no prob with you writing code in this style. a future employer might have you write like the example above.